### DISCUSSION TITLE
Try SEAIT for One-Click Installation and Space-Saving Model Management.

### DISCUSSION THREAD

**Discussion by diStyApps** [anapnoe/stable-diffusion-webui-ux#105] [Show and tell]
Thank you for this fork, cool ui.

Check seait for easier installation and space saving model management.
https://github.com/diStyApps/seait
https://civitai.com/models/27574/super-easy-ai-installer-tool

https://user-images.githubusercontent.com/3084832/235312746-2f2be858-8814-4d12-b6a4-2cf8113985d4.mp4

**Comment by anapnoe** [comment]
cool this is great and useful

**Reply by diStyApps** [reply]
Thank you very much i appreciate it.
